metagraph/docs/source/api.rst

```diff
@@ -1,7 +1,7 @@
 .. _api:
 
-API
-===
+Python API
```

Comment:
Currently, there is just a list of anecdotal examples being provided. Is there a more complete documentation of the Python API that we could point to from here? Otherwise, maybe a basic documentation of the API should be added here.

Reply:
Yes, this is all we have for now.